### PR TITLE
Add default option to download the small sized Open Bandit Dataset with the dataset module

### PR DESCRIPTION
**Pull request by usaito**
## Overview
In the current version, we have to clone the repository to use the example obd data.
It is awkward and, in this PR, I want to add a default option to download the small sized Open Bandit Dataset to the dataset module as follows.

### Before
We have to first clone the repository and then specify the correct `data_path` to use the example obd.

スクリーンショット 2021-02-05 6 25 40

### After

We do not have to specify a particular `data_path` to use the example obd because it is now contained in the dataset module.

スクリーンショット 2021-02-05 6 24 55

ref: https://github.com/st-tech/zr-obp/blob/add-obd-dl-option/examples/quickstart/obd.ipynb

## Review points
- [obp/dataset/real](https://github.com/st-tech/zr-obp/blob/add-obd-dl-option/obp/dataset/real.py)
https://github.com/st-tech/zr-obp/blob/406eb832bac34a51037a343c1d81ac8b4d72a673/obp/dataset/real.py#L50-L71

- [examples/obd](https://github.com/st-tech/zr-obp/tree/add-obd-dl-option/examples/obd)
- [examples/quickstart/obd.ipynb](https://github.com/st-tech/zr-obp/blob/add-obd-dl-option/examples/quickstart/obd.ipynb)

### CODE REVIEW

**Comment by usaito** [comment]
@nmasahiro Can you review this PR?

**Comment by nmasahiro** [comment]
@usaito Sure! I leave some minor comments.

File: obp/dataset/real.py [hunk 1]
```diff
@@ -34,7 +34,8 @@ class OpenBanditDataset(BaseRealBanditDataset):
         One of the three possible campaigns considered in ZOZOTOWN, "all", "men", and "women".
 
     data_path: Path, default=Path('./obd')
-        Path that stores Open Bandit Dataset.
+        Path where the Open Bandit Dataset exists.
+        When `data_path` is not given, this class downloads the example small sized version of the dataset.
```
Comment (nmasahiro) [inline obp/dataset/real.py:38]: [imo] I think that `small-sized` is more appropriate than `small sized`.

**Comment by nmasahiro** [comment]
[imo]
When using a small sample, I think it's better to show a message that makes it clear that users are using small data.
What do you think?

**Comment by nmasahiro** [comment]
[nits] Can you execute a `black` command?
I found some inconsistency in my local laptop when executing `black` command.

**Comment by usaito** [comment]
@nmasahiro  

Thank you for the review!
I incorporated all your comments and appreciate your re-check! 

スクリーンショット 2021-02-06 23 37 15（2）

**Pull request merged**